docs/source/index.md

```diff
@@ -10,6 +10,19 @@ in JupyterLab and the Jupyter Notebook. More specifically, Jupyter AI offers:
 * Support for a wide range of generative model providers and models
   (AI21, Anthropic, Cohere, Hugging Face, OpenAI, SageMaker, etc.).
 
+## JupyterLab support
+
+**Each major version of Jupyter AI supports *only one* major version of JupyterLab.** Jupyter AI 1.x supports
+JupyterLab 3.x, and Jupyter AI 2.x supports JupyterLab 4.x. The feature sets of versions 1.0.0 and 2.0.0
+are the same. We will maintain support for JupyterLab 3 for as long as it remains maintained; that is,
+until JupyterLab 5.0.0 is available.
+
+The `main` branch of Jupyter AI targets the newest supported major version of JupyterLab, which, as of
+July 2023, is version 4. All new features and most bug fixes will be
```

Comment:
```suggestion
The `main` branch of Jupyter AI targets the newest supported major version of JupyterLab. All new features and most bug fixes will be
```

ditto, no need to provide a timestamped declaration of JupyterLab's latest major version when it's easily discoverable and well-known.